packages/sdk-js/src/GrowthBook.ts

```diff
@@ -926,7 +1010,8 @@ export class GrowthBook<
       return this._getResult(experiment, -1, false, featureId);
     }
 
-    // 9. Get bucket ranges and choose variation
+    // 9. Get the variation from the sticky bucket or get bucket ranges and choose variation
+    let ranges: VariationRange[] = [];
```

Comment:
Is this needed here?  Can we move this within the `if (!foundStickyBucket)` block below?